src/components/protocol_handler/include/protocol_handler/handshake_handler.h

```diff
@@ -89,6 +89,11 @@ class HandshakeHandler : public security_manager::SecurityManagerListener {
       uint32_t connection_key,
       security_manager::SSLContext::HandshakeResult result) OVERRIDE;
 
+  /**
+   * @brief Notification about handshake failure
+   */
+  void OnHandshakeFailed() OVERRIDE;
```

Comment:
@AByzhynar would be nice to describe what exactly will be done OnHandshakeFailed

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad